### PR TITLE
[DOC-3072] Update Delegate overview topic

### PR DESCRIPTION
**Pull request by brian-f-harness**
Applied standards, fixed formatting

For reviewers, [a preview](https://6453c92c979c2f3c5697039c--harness-developer.netlify.app/docs/platform/Delegates/delegate-concepts/delegate-overview) is available.

## What Type of PR is This?

- [ ] Issue
- [ ] Feature
- [x] Maintenance/Chore

If tied to an Issue, list the Issue(s) here:

* *Issue(s)*

## House Keeping
Some items to keep track of. Screen shots of changes are optional but would help the maintainers review quicker. 

- [x] Tested Locally
- [ ] *Optional* Screen Shoot.

### CODE REVIEW

File: docs/platform/2_Delegates/delegate-concepts/delegate-overview.md [hunk 1]
```diff
@@ -66,68 +66,70 @@ One Delegate size does not fit all use cases, so Harness let's you pick from sev
...
 
 ### Delegate high availability (HA)
 
-You might need to install multiple Delegates depending on how many Continuous Delivery tasks you do concurrently, and on the compute resources you are providing to each Delegate. Typically, you will need one Delegate for every 300-500 service instances across your applications.
+You might need to install multiple delegates depending on how many Continuous Delivery tasks you do concurrently, and on the compute resources you are providing to each delegate. Typically, you will need one delegate for every 300-500 service instances across your applications.
 
-In addition to compute considerations, you can enable High Availability (HA) for Harness Delegates. HA simply involves installing multiple Delegates in your environment.
+In addition to compute considerations, you can enable High Availability (HA) for Harness Delegates. HA involves installing multiple delegates in your environment.
```
Comment (StrawberryPad) [inline docs/platform/2_Delegates/delegate-concepts/delegate-overview.md:127]:
```suggestion
In addition to compute considerations, you can enable HA for Harness Delegates. HA involves installing multiple delegates in your environment.
```

File: docs/platform/2_Delegates/delegate-concepts/delegate-overview.md [hunk 2]
```diff
@@ -66,68 +66,70 @@ One Delegate size does not fit all use cases, so Harness let's you pick from sev
...
 
-For example, your Kubernetes deployment could include two Kubernetes Delegates, each running in its own pod in the same target cluster. To add Delegates to your deployment, increase the desired count of Delegate replica pods in the **spec** section of the harness-kubernetes.yaml file that you download from Harness:
+For example, your Kubernetes deployment could include two Kubernetes delegates, each running in its own pod in the same target cluster. To add delegates to your deployment, increase the desired count of delegate replica pods in the **spec** section of the harness-kubernetes.yaml file that you download from Harness:
```
Comment (StrawberryPad) [inline docs/platform/2_Delegates/delegate-concepts/delegate-overview.md:129]:
```suggestion
For example, your Kubernetes deployment could include two Kubernetes delegates, each running in its own pod in the same target cluster. 

To add delegates to your deployment, increase the desired count of delegate replica pods in the **spec** section of the harness-kubernetes.yaml file that you download from Harness:
```

File: docs/platform/2_Delegates/delegate-concepts/delegate-overview.md [hunk 3]
```diff
@@ -145,45 +147,44 @@ spec:
...
+In every case, delegates must be identical in terms of permissions, keys, connectivity, and so on. With two or more delegates running in the same target environment, you get HA by default. One delegate can go down without impacting Harness' ability to perform deployments. If you want more availability, you can set up three delegates to handle the loss of two delegates, and so on.
 
-Two delegates in different locations with different connectivity do not support high availability. For example, if you have a delegate in a Dev environment and another in a Prod environment, there is no communication between the two delegates. If either delegate fails, Harness stops operating in that environment.
+Two delegates in different locations with different connectivity do not support HA. For example, if you have a delegate in a Dev environment and another in a Prod environment, there is no communication between the two delegates. If either delegate fails, Harness stops operating in that environment.
 
 ### Delegate scope
 
 Delegates are scoped in two ways:
```
Comment (StrawberryPad) [inline docs/platform/2_Delegates/delegate-concepts/delegate-overview.md:161]: I only see one way below.

File: docs/platform/2_Delegates/delegate-concepts/delegate-overview.md [hunk 4]
```diff
@@ -8,27 +8,27 @@ helpdocs_is_private: false
...
 
-* **Heartbeat** - The Delegate sends a [heartbeat](https://en.wikipedia.org/wiki/Heartbeat_(computing)) to notify Harness Manager that it is running.
-* **Deployment data** - The Delegate sends information retrieved from API calls to Harness Manager for display on the **Deployments** page.
-* **Time series and log data for Continuous Verification** - The Delegate connects to the verification providers you configure and sends the data retrieved from those providers to Harness Manager for display in Harness Continuous Verification.
+* **Heartbeat:** The delegate sends a [heartbeat](https://en.wikipedia.org/wiki/Heartbeat_(computing)) to notify Harness Manager that it is running.
+* **Deployment data:** The delegate sends information retrieved from API calls to Harness Manager for display on the **Deployments** page.
+* **Time series and log data for Continuous Verification:** The delegate connects to the verification providers you configure and sends the data retrieved from those providers to Harness Manager for display in Harness Continuous Verification.
```
[the remark this answers is not in the document]
Comment (brian-f-harness) [reply docs/platform/2_Delegates/delegate-concepts/delegate-overview.md:26]: Adding with a commit:

Delegate communication includes the following functions:

File: docs/platform/2_Delegates/delegate-concepts/delegate-overview.md [hunk 5]
```diff
@@ -37,27 +37,27 @@ Harness Delegate does not have a root image. There are two non-root images that
 * `harness/delegate:22.03.74411`
 * `harness/delegate:22.03.74411.minimal`
 
-The first image includes client tools like `kubectl`, Helm, and ChartMuseum. The second image, for which the `minimal` tag is appended, does not include those client tools. If you want to add tools to the image, Harness recommends the creation of a custom image.
+The first image includes client tools like `kubectl`, Helm, and ChartMuseum. The second image, for which the `minimal` tag is appended, does not include those client tools. If you want to add tools to the image, Harness recommends that you create a custom image.
```
[the remark this answers is not in the document]
Comment (brian-f-harness) [reply docs/platform/2_Delegates/delegate-concepts/delegate-overview.md:40]: Great suggestion. Added the following:

* `harness/delegate:22.03.74411`: Includes client tools like `kubectl`, Helm, and ChartMuseum. 
* `harness/delegate:22.03.74411.minimal`: Does not include client tools. If you want to add tools to the image, Harness recommends that you create a custom image.

File: docs/platform/2_Delegates/delegate-concepts/delegate-overview.md (hunk 3 above)
Comment (brian-f-harness) [reply docs/platform/2_Delegates/delegate-concepts/delegate-overview.md:161]: Nice catch

**Pull request merged**